client-core/Cargo.toml

```diff
@@ -12,11 +12,11 @@ client-index = { path = "../client-index" }
 secp256k1zkp = { git = "https://github.com/crypto-com/rust-secp256k1-zkp.git", rev = "4c178ca90965ccf5b1b235e346e836ca65b2a549", features = ["serde", "zeroize", "rand"] }
 rand = "0.6"
 failure = "0.1"
-bincode = "1.1"
 hex = "0.3"
 zeroize = "0.6"
 byteorder = "1.3"
-rlp = "0.3"
+parity-codec = "3.5.1"
+parity-codec-derive = "3.3.0"
```

Comment:
Remove `parity-codec-derive` dependency and add `"derive"` feature in `parity-codec`.